deployment/terraform/assets/cluster.tf

```diff
@@ -572,6 +572,16 @@ resource "aws_security_group_rule" "metrics-pyroscope" {
   security_group_id = aws_security_group.metrics[0].id
 }
 
+resource "aws_security_group_rule" "metrics-loki" {
+  count             = var.app_instance_count > 0 ? 1 : 0
```

Comment:
Conceptually, I'd expect Loki to be needed as long as there's a metrics instance.

Reply:
That makes sense, let me check that instead